src/main/resources/com/mathworks/ci/RunMatlabBuildBuilder/help-tasks.html

```diff
@@ -0,0 +1,11 @@
+<div>
+    Insert a space-separated list of tasks to run in the <b>Tasks</b> box. If not specified, the action runs the default tasks in buildfile.m as well as all the tasks on which they depend.<br>
+    <b>Example: </b>test<br>
+    <b>Example: </b>compile test
+    <br>&nbsp;</br>
+    <b>Note:</b>
+    <ul>
+        <li>MATLAB exits with exit code 0 if the tasks run without error. Otherwise, MATLAB terminates with a nonzero exit code, which causes the workflow to fail.</li>
```

Comment:
Does it make sense to replace workflow with build.

Reply:
Yes good catch